marketplace_builder/pages/getting-started/notifications/notifications.liquid

```diff
@@ -0,0 +1,48 @@
+---
+converter: markdown
+metadata:
+  title: Notifications
+slug: get-started/notifications/notifications
+---
+
+**Notifications** are messages sent to Platform OS users (including admins) when something happens. A message can be an email, SMS, or programmatic call to a 3rd party API.
+
+Notifications can be delayed, and you can use Liquid, GraphQL, and trigger conditions to decide if a notification should be sent. They are a powerful mechanism used for example to welcome new users, follow up after they've added their first item, or reach out to them if they have been inactive for a while. 
+
+## Supported notification types
+
+Currently, we support three types of notifications:
+
+* Email
+* SMS
+* API call
+
+## Notifications directory
+
+All notifications are defined in the `notifications` directory.  
+
+Depending on what type of the notification you would like to define, you should have three subdirectories:
+
+* `email_notifications`: contains all email notifications
+*  `sms_notifications`: contains all sms notifications
+*  `api_call_notifications`: contains all API call notifications
+
+## Configuration 
+
+Each configuration can accept different properties, but some of the main concepts are common:
+
+| Property            | Description                                                                                                                                                                                                        | Default | Example                                                                                                                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `to`                | Depending on the alert type, it is either: {::nomarkdown}<ul><li>an email address of the recipient</li><li>mobile number of the recipient</li><li>endpoint for the API call</li></ul>{:/} Accepts liquid.          | n/a     | `test@email.com`                                                                                                                                                             |
```

Comment:
`{::nomarkdown}` will need to be verified - im positive its available in jekyll, buti im not sure its available in our markdown implementation

Reply:
@piotrze Could you please verify if `{::nomarkdown}` is available in our markdown implementation? Thanks!